### ISSUE TITLE
Regional Backend Service does not appear to be supported

### ISSUE THREAD

**Issue by rwkarg** [GoogleCloudPlatform/gke-autoneg-controller#23]
This is required to use against an Internal HTTP(S) Load Balancer.

**Comment by rwkarg** [comment]
`telemetry-gateway-qa-us-http` is a regional backend service in an Internal HTTPS Load Balancer

```
2021-01-27T00:58:44.985Z INFO controllers.Service Applying intended status {"service": "europa-us/telemetry-gateway", "status": {"name":"telemetry-gateway-qa-us-http","max_rate_per_endpoint":100,"network_endpoint_groups":{"8443":"k8s1-28548957-europa-us-telemetry-gateway-8443-e31d0e4f"},"zones":["us-west2-b"]}}

2021-01-26 16:58:45.351 PST2021-01-27T00:58:45.351Z ERROR controller-runtime.controller Reconciler error {"controller": "service", "request": "europa-us/telemetry-gateway", "error": "backend service not found"}
```

**Comment by soellman** [comment]
Fixed by #34
